src/openlifu/io/tx7332_if.py

```diff
@@ -0,0 +1,372 @@
+import logging
+import struct
+from typing import List
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Setting the root logger to debug level at the top of this file is not appropriate as this will affect logging globally just when the file gets imported. I will adjust this.